src/diffusers/pipelines/controlnet/pipeline_controlnet.py

```diff
@@ -1146,7 +1147,7 @@ def __call__(
             )
 
         # 4. Prepare image
-        if isinstance(controlnet, ControlNetModel):
+        if isinstance(controlnet, ControlNetModel) and image:
```

Comment:
We try to be as explicit as possible with this. So:

`if isinstance(controlnet, ControlNetModel) and image is not None:`